nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -61,6 +61,18 @@
       This is to make it possible to use <literal>podman</literal> instead of <literal>docker</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      MariaDB has been updated to 10.4, MariaDB Galera to 26.4.
+      Before you upgrade, it would be best to take a backup of your database.
+      For MariaDB Galera Cluster, see <link xlink:href="https://mariadb.com/kb/en/upgrading-from-mariadb-103-to-mariadb-104-with-galera-cluster/">Upgrading
+      from MariaDB 10.3 to MariaDB 10.4 with Galera Cluster</link> instead.
+      Before doing the upgrade read <link xlink:href="https://mariadb.com/kb/en/upgrading-from-mariadb-103-to-mariadb-104/#incompatible-changes-between-103-and-104">Incompatible
```

Comment:
It might be worth explicitly stating (in bold) the user will need to run `mysql_upgrade` after upgrading, even though this is included in upstream documentation. Since NixOS is not entirely consistent about running migrations some users may assume this was already done for them.

Reply:
Updated release note and rebased PR.